pyproject.toml

```
@@ -28,7 +28,7 @@ dev = [
 ]
 test = [
     "pre-commit",
-    "pytest",
+    "pytest >=9.16",
```

I think this is a typo?
```suggestion
    "pytest",
```